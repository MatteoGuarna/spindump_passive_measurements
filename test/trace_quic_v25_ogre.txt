A test with ATS (Ogre) QUIC implementation server and Quant client.

There are four sessions:

- regular + full start
- regular + 0rtt
- key update
- quantum-mode

The commands used were:

   rm /tmp/quant*
   ./bin/client -i en0 https://quic.ogre.com:4433/15000 
   ./Bin/client -i en0 https://quic.ogre.com:4433/15000 
   ./bin/client -u -i en0 https://quic.ogre.com:4433/15000 
   ./bin/client -m -i en0 https://quic.ogre.com:4433/15000 

