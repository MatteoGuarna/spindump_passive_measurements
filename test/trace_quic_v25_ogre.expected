QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d f506d2a43be42e283a963b63a7d80104b168-null (58496:4433) at 1580815317204706 new starting packets 0 0 bytes 0 0 note "No rsp,v.0xbabababa,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d f506d2a43be42e283a963b63a7d80104b168-b16806b40ff506 (58496:4433) at 1580815317346032 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v.0x00000000,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d f506d2a43be42e283a963b63a7d80104b168-b16806b40ff506 (58496:4433) at 1580815317346032 measurement starting right 141326 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v.0x00000000,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d null-b16806b40ff506 (58496:4433) at 1580815317347637 measurement starting left 1605 packets 1 1 bytes 1248 82 bandwidth 1248 82 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d null-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317492683 change starting packets 2 1 bytes 2496 82 bandwidth 2496 82 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d null-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317492683 measurement up right 145046 packets 2 1 bytes 2496 82 bandwidth 2496 82 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317492689 packet up responder length 1231 packets 2 3 bytes 2496 2615 bandwidth 2496 2615 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317493955 packet up responder length 1138 packets 2 4 bytes 2496 3753 bandwidth 2496 3753 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317501141 packet up initiator length 102 packets 3 4 bytes 2598 3753 bandwidth 2598 3753 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317501444 packet up initiator length 101 packets 4 4 bytes 2699 3753 bandwidth 2699 3753 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317501855 packet up initiator length 101 packets 5 4 bytes 2800 3753 bandwidth 2800 3753 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317639659 packet up responder length 1231 packets 5 5 bytes 2800 4984 bandwidth 2800 4984 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317639667 packet up responder length 1231 packets 5 6 bytes 2800 6215 bandwidth 2800 6215 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317640526 packet up initiator length 101 packets 6 6 bytes 2901 6215 bandwidth 2901 6215 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317640783 packet up responder length 344 packets 6 7 bytes 2901 6559 bandwidth 2901 6559 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317642249 packet up initiator length 1500 packets 7 7 bytes 4401 6559 bandwidth 4401 6559 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 861ebbb7-12cb72218bea5ebb92d4eedd15c911cc61ee (58496:4433) at 1580815317783274 packet up responder length 379 packets 7 8 bytes 4401 6938 bandwidth 4401 6938 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04d2e890-13d4261e4a744f8afeb6931585f716ee8ea8cbc3 (58497:4433) at 1580815321457759 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04d2e890-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321602637 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04d2e890-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321602637 measurement up right 144878 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d d2e890d6-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321605615 packet up initiator length 164 packets 2 1 bytes 1412 549 bandwidth 1412 549 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d d2e890d6-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321744509 packet up responder length 529 packets 2 2 bytes 1412 1078 bandwidth 1412 1078 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d d2e890d6-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321746579 packet up initiator length 1500 packets 3 2 bytes 2912 1078 bandwidth 2912 1078 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d d2e890d6-12f54c12a6fc06f144831143208b2d07c232 (58497:4433) at 1580815321885365 packet up responder length 128 packets 3 3 bytes 2912 1206 bandwidth 2912 1206 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04e0355e-25f5058be3709f6c52edde (58498:4433) at 1580815326233625 new starting packets 0 0 bytes 0 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04e0355e-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326379646 change starting packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04e0355e-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326379646 measurement up right 146021 packets 1 0 bytes 1248 0 bandwidth 1248 0 note "No rsp,v25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d e0355ebc-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326382589 packet up initiator length 164 packets 2 1 bytes 1412 549 bandwidth 1412 549 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d e0355ebc-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326524745 packet up responder length 530 packets 2 2 bytes 1412 1079 bandwidth 1412 1079 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d e0355ebc-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326526810 packet up initiator length 1500 packets 3 2 bytes 2912 1079 bandwidth 2912 1079 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d e0355ebc-120dd5ede0e69213c48d8ed3c924dc0bb0dd (58498:4433) at 1580815326664961 packet up responder length 130 packets 3 3 bytes 2912 1209 bandwidth 2912 1209 note "V25,0-RTT,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04955e46-9ce0e53a8214e51e565589c19952bd59 (58500:4433) at 1580815330609900 new starting packets 0 0 bytes 0 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04955e46-9ce0e53a8214e51e565589c19952bd59 (58500:4433) at 1580815330609901 packet starting initiator length 637 packets 2 0 bytes 1917 0 bandwidth 1917 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04955e46-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815330748460 change starting packets 2 0 bytes 1917 0 bandwidth 1917 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 04955e46-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815330748460 measurement up right 138559 packets 2 0 bytes 1917 0 bandwidth 1917 0 note "No rsp,v25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 955e46aa-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815330751054 packet up initiator length 164 packets 3 1 bytes 2081 549 bandwidth 2081 549 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 955e46aa-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815330881687 packet up responder length 529 packets 3 2 bytes 2081 1078 bandwidth 2081 1078 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 955e46aa-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815330883577 packet up initiator length 1500 packets 4 2 bytes 3581 1078 bandwidth 3581 1078 note "V25,no spin"
QUIC 2a00:79e1:abc:301:18d2:7b31:c60c:74c6 <-> 2001:4800:7817:101:be76:4eff:fe04:631d 955e46aa-12fbb8878975e18ce5552fe9cc399ee1bc69 (58500:4433) at 1580815331014608 packet up responder length 129 packets 4 3 bytes 3581 1207 bandwidth 3581 1207 note "V25,no spin"
